citi-onboarding/maracatronics
# feature(collaborators-description): create collaborators description …

## What i did
- Created the collaborators description
- Delete the Post model

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/collaborators-description" with ```git checkout feature/collaborators-description```
3. Now, you need to found the server folder with ```cd server```
4. On the server folder, run the server with ```node index.js``` and go to http://localhost:3001/admin
5. Go to "Parceiros" model and create a new collaborator
6. Check if there's the description input 
7. Now, go to the route: http://localhost:3001/api/parceiros
8. On that route, check if there are the description part and the collaborator you created
9. Don't forget to delete the collaborator you created
